Runtime/DataModels/TaskResult.cs

```diff
@@ -0,0 +1,32 @@
+using Newtonsoft.Json.Linq;
+
+namespace ReupVirtualTwin.dataModels
+{
+    public class TaskResult
+    {
+        public bool IsSuccess { get; }
```

Comment:
```suggestion
        public bool isSuccess { get; }
```
I would expect these to be lower case camel, but I don't know all of the conventions in c#